File: 프로그래머스/1/133499. 옹알이 （2）/옹알이 （2）.py
def solution(babbling):
    answer = 0
    words = ["aya", "ye", "woo", "ma"]

    for v in babbling:
        for w in words:
            if w*2 in v: break;
            v = v.replace(w, " ")
        if v.strip() == "": answer += 1
    return answer

File: 프로그래머스/1/133499. 옹알이 （2）/옹알이 （2）.js
function solution(babbling) {
    let answer = 0;
    const words = ["aya", "ye", "woo", "ma"];
    
    for (let v of babbling) {
        for (let w of words) {
            if (v.includes(w.repeat(2))) break;
            v = v.split(w).join(" ");
        }
        if (v.split(" ").join("") === "") answer++
    }
    return answer;
}